app/src/main/java/com/mashup/lastgarden/ui/scent/comment/ScentCommentAdapter.kt

```diff
@@ -0,0 +1,61 @@
+package com.mashup.lastgarden.ui.scent.comment
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import com.mashup.lastgarden.R
+import com.mashup.lastgarden.data.vo.Comment
+import com.mashup.lastgarden.databinding.ItemCommentBinding
+import com.mashup.lastgarden.extensions.btnThumbsUpDownSelector
+
+class ScentCommentAdapter(
+    private val list: List<Comment>,
+    private val listener: OnClickListener? = null
+) : RecyclerView.Adapter<ScentCommentAdapter.ScentCommentViewHolder>() {
+
+    override fun onCreateViewHolder(
+        parent: ViewGroup,
+        viewType: Int
+    ): ScentCommentViewHolder {
+        val view = LayoutInflater.from(parent.context)
+            .inflate(R.layout.item_comment, parent, false)
+        return ScentCommentViewHolder(ItemCommentBinding.bind(view))
```

Comment:
ViewBinding을 사용하면 LayoutInflater 사용할 필요가 없을 것 같아요.
```suggestion
        return ScentCommentViewHolder(
                 ItemCommentBinding.inflate( LayoutInflater.from(parent.context), parent, false)
        )
```